src/compiler/crystal/util.cr

```diff
@@ -4,7 +4,7 @@ module Crystal
     if filename.starts_with?(dir)
       filename = filename[dir.size..-1]
       if filename.starts_with? '/'
-        filename[1..-1]
+        filename.lchop
```

Comment:
```suggestion
        filename.lchop('/')
```

ditto below

Reply:
Actually the more unspecific `lchop` without a char is faster:
```
lchop(char)  30.81M ( 32.46ns) (±11.64%)  32 B/op   1.21× slower
      lchop  37.38M ( 26.75ns) (±15.55%)  32 B/op        fastest
```
Also, I don't think there's really a readability difference. If the filename starts with `/`, remove that from the beginning.

Reply:
With `lchop('/')` you can spare the entire if clause. That should result in pretty much the same instructions and more concise code.

Reply:
Ooh yes overlooked that the `if` can just be removed then. Will change that. But for the `lchop` below I don't even think I'm operating on a filename so `lchop('/')` there would probably be wrong. Will leave that one as-is.